optax/_src/wrappers_test.py

```diff
@@ -262,10 +262,10 @@ class MaskedTest(chex.TestCase):
 
   @chex.all_variants
   @parameterized.named_parameters(
-      ('scale', lambda: transform.scale(-1.), True),  # stateless test
-      ('sgd', _build_sgd, False),  # stateful test
-      ('scale', lambda: transform.scale(-1.), True),  # stateless test
-      ('sgd', _build_sgd, False),  # stateful test
+      ('sgd', _build_sgd, False),
+      ('stateful_sgd', _build_stateful_sgd, False),
+      ('sgd_w_fn', _build_sgd, True),
```

Comment:
Nit: since this is the name that will be displayed when the test fails I'd write this out to `sgd_with_mask_fn` or even `sgd_with_mask_function` (and similarly in the line below).